README.md

```diff
@@ -181,8 +368,11 @@ axe-matchers supports Capybara, Selenium, and Watir webdrivers; each tested with
 [rules-clause]: #checking-rules-clause
 [exclusive-rules-clause]: #exclusive-rules-clause
 [skipping-rules-clause]: #skipping-rules-clause
+[configure browser webdriver]: #configuration
 
 [a11ycheck]: https://github.com/dequelabs/axe-core/blob/master/doc/API.md#api-name-axea11ycheck
 [context-param]: https://github.com/dequelabs/axe-core/blob/master/doc/API.md#a-context-parameter
 [options-param]: https://github.com/dequelabs/axe-core/blob/master/doc/API.md#b-options-parameter
 [rules]: https://github.com/dequelabs/axe-core/blob/master/doc/rule-descriptions.md
+
+[css selector]: http://www.w3schools.com/cssref/css_selectors.asp
```

Comment:
I can't, in good conscience, link to w3schools. But I'm cool with MDN: https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Getting_started/Selectors